tools/automate_all_tests.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+set -x
+
+USAGE="
+Usage: automate_all_tests.sh
+  --key_file <path to key file>
+  --branch <branch to checkout and build>
+  --env_path <path to GCE compatible environment directory>
+"
+
+while true; do
+  case "$1" in
+  --branch)
+    BRANCH=$2
+    shift 2
+    ;;
+  --key_file)
+    KEY_FILE=$2
+    shift 2
+    ;;
+  --env_path)
+    ENV_PATH=$2
+    shift 2
+    ;;
+  --)
+    shift
+    break
+    ;;
+  *) break ;;
+  esac
+done
+
+if [[ -z "${BRANCH}" ]] || [[ -z "${KEY_FILE}" ]] || [[ -z "${ENV_PATH}" ]]; then
+  echo "${USAGE}"
+  exit 1
+fi
+
+ROOT_DIR=$(git rev-parse --show-toplevel)
+DATA_STEWARD_DIR="${ROOT_DIR}/data_steward"
+DEID_DIR="${DATA_STEWARD_DIR}/deid"
+export GH_USERNAME=$(whoami)
```

Comment:
I _don't think_ this would ever be the GitHub username..?
```suggestion
export CURRENT_USER=$(whoami)
```

Reply:
You're right.  `GH_USERNAME` had to be set to run `curation/data_steward/init.sh` correctly.  It requires a `GH_USERNAME` or `CIRCLE_USERNAME`.

Reply:
Ah, this is ok then

Reply:
I'll add a comment in here to explain why this was set this way.  This script should primarily be used for automating on the VM.  I strongly discourage anyone use this script to run the tests on their own.